src/test/java/org/apache/sling/resourceresolver/impl/ResourceResolverFactoryTest.java

```diff
@@ -108,7 +109,7 @@
     @Test public void testGetAllowedAliasPaths() throws NoSuchMethodException {
         assertTrue(this.commonFactory.getAllowedAliasPaths().isEmpty());
         String[] allowPaths = {"/parent", "/parent0"};
-        setInaccessibleField("aliasPathAllowList", activator, Collections.unmodifiableList(Arrays.asList(allowPaths)));
+        setInaccessibleField("aliasPathAllowList", activator, Collections.unmodifiableSet(new HashSet<>(Arrays.asList(allowPaths))));
```

Comment:
I would use `TreeSet` again instead of `HashSet`, even in the test. Just in case some logic (now or in the future) relies on the allowed alias locations to be sorted.